controllers/suite_test.go

```diff
@@ -86,13 +112,19 @@ var _ = BeforeSuite(func() {
 	err = ocmclv1.AddToScheme(scheme.Scheme)
 	Expect(err).NotTo(HaveOccurred())
 
-	err = subv1.AddToScheme(scheme.Scheme)
+	err = plrv1.AddToScheme(scheme.Scheme)
 	Expect(err).NotTo(HaveOccurred())
 
-	err = ramendrv1alpha1.AddToScheme(scheme.Scheme)
+	err = viewv1beta1.AddToScheme(scheme.Scheme)
 	Expect(err).NotTo(HaveOccurred())
 
-	err = viewv1beta1.AddToScheme(scheme.Scheme)
+	err = cpcv1.AddToScheme(scheme.Scheme)
```

Comment:
Following are added for Policy and PlacementBinding use with secrets to managed clusters. Helps adding them with this PR, to address imports in one go.